draft-ietf-anima-brski-prm.md

```diff
@@ -155,34 +158,29 @@ BRSKI addresses scenarios in which the pledge initiates the bootstrapping acting
+
 * introduces the Registrar-Agent as new component to facilitate the communication between the pledge and the domain registrar.
   The Registrar-Agent may be implemented as an integrated functionality of a commissioning tool or be co-located with the domain registrar itself.
   BRSKI-PRM supports the identification of the Registrar-Agent that was performing the bootstrapping allowing for accountability of the pledges installation, when the Registrar-Agent is a component used by an installer and not co-located with the domain registrar.
 
-* specifies the interaction (data exchange and data objects) between a pledge acting as server, the Registrar-Agent acting as client, and the domain registrar.
+* specifies additional artifacts for the exchanges between a pledge acting as server, the Registrar-Agent acting as client, and the domain registrar acting as server toward the Registrar-Agent.
 
-* enables the usage of arbitrary transports between the pledge and the domain registrar via the Registrar-Agent; security is addressed at the application layer, and both IP-based and non-IP connectivity can be used between pledge and Registrar-Agent.
+* allows the application of Registrar-Agent credentials to establish TLS connections to the domain registrar; these are different from the pledge IDevID credentials.
 
-* allows the application of Registrar-Agent credentials to establish TLS connections to the domain registrar; these are different from the IDevID of the pledge.
+* also enables the usage of alternative transports, both IP-based and non-IP, between the pledge and the domain registrar via the Registrar-Agent; security is addressed at the application layer through an additional signature wrapping of the exchanged artifacts.
 
 The term endpoint used in the context of this document is equivalent to resource in HTTP {{RFC9110}} and CoAP {{RFC7252}}; it is not used to describe a device.
 Endpoints are accessible via Well-Known URIs {{RFC8615}}.
-For the interaction with the domain registrar, the Registrar-Agent will use existing BRSKI {{!RFC8995}} endpoints as well as additional endpoints defined in this document.
-To utilize the EST server endpoints on the domain registrar, the Registrar-Agent will act as client toward the registrar.
-
-The Registrar-Agent also acts as a client when communicating with a pledge that is in responder mode.
-Here, TLS with server-side, certificate-based authentication is only optionally supported.
-If TLS is optionally used between the Registrar-Agent and the pledge, the Registrar-Agent needs to identify the pledge based on its product-serial-number rather than the hostname, as the latter is not set in an IDevID certificate.
-
-BRSKI-PRM is designed to rely on object security to support also for alternative transports for which TLS may not be available, e.g., Bluetooth or NFC.
-This is achieved through an additional signature wrapping of the exchanged data objects involving the Registrar-Agent for transport.
 
```

Comment:
was this considered as too detailed information for the introduction?

Reply:
Yes, the TLS aspect is discussed at length later and a bit out-of-context in the introduction.

The object security part is already covered in the last bullet point. We could keep that keyword by going for

> security is addressed at the application layer through object security with an additional signature wrapping the exchanged artifacts

Reply:
yes, would be perfect.